modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -1103,6 +1107,18 @@ public void incrementViewCount(
 			increment);
 	}
 
+	@Override
+	public Lock lock(long userId, long resourcePrimKey) throws PortalException {
+		if (!FeatureFlagManagerUtil.isEnabled("LPS-195016")) {
+			return null;
+		}
+
+		return LockManagerUtil.lock(
```

Comment:
Why not inject the `LockManager` service instead of using the static util?

Reply:
Requested change done in new commit [c959d71](https://github.com/liferay-lima/liferay-portal/pull/4991/commits/c959d71436eee5ca124c1730af0cc303181d60ce)